data/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.7-slim-stretch
+
+RUN mkdir -p /usr/local/src
+WORKDIR /usr/local/src
+
+COPY requirements.txt /usr/local/src/
+
+RUN set -ex \
+	&& buildDeps=" \
+		build-essential \
+	" \
+	&& deps=" \
+		unzip \
+		make \
+		wget \
```

Comment:
We don't need these dependencies immediately, but we'll probably need them once we start collecting shapefiles.